packages/docs/src/helpers/CodeSandboxIconsHelper.ts

```diff
@@ -0,0 +1,68 @@
+
+export const iconsStyles = `
+  <link
```

Comment:
This part probably needs some polish.

We want to provide some comments on which part does what. i.e.

```html
<!-- Font Awesome 4 -->
```

- [ ] We want to keep only one version of font-awesome.
- [ ] For ionicons might make sense to update to latest version (current one is 5.*)

You might want to move that into another issue.

Reply:
@asvae upgraded the Ionicon version.